tests/integration/create-switch-account.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { BrowserDriver, createTestSelector, setupBrowser } from './utils';
```

Comment:
Should this just be moved into the `settings.spec.ts`. It seems like it should go in with those tests?

Reply:
I am fine with either. I just did not want to crowd the settings.spec.ts

Reply:
I am not sure what is typical for integration tests but if is better to break them all out separately, I am totally fine with that.

Reply:
I think it's fine to break them out. It's fine to group to begin with, but eventually when you have a 500 line file, definitely helpful to have broken out